lib/KnightLab_Sensors/src/KL_ADAFRUIT_SI7021.cpp

```diff
@@ -18,14 +25,34 @@ namespace ADAFRUIT_SI7021 {
         }
     }
 
-    float readTemperature()
+    bool start()
+    {
+        return true;
+    }
+
+    size_t read(char* buf, int len)
+    {
+        float temp = sensor.readTemperature() * 1.8 + 32;
```

Comment:
is this compiling. Should be pulling readTemperature and readHumidity from the local namespace of this driver